Plugin/Framework/Search/Request/AbstractHandler.php

```diff
@@ -201,11 +191,7 @@ private function setFallbackSort(array &$requestData)
                 'direction' => 'ASC'
             ];
         } catch (Exception $e) {
-            $this->logger->debugWithSource(
-                sprintf("Could not set fallback sorting. %s", $e->getMessage()),
-                [],
-                $this
-            );
+            $this->exception($e);
```

Comment:
@olsi-qose we need to sanity-check this.

Reply:
Done! 👍